crates/ruff_server/tests/snapshots/notebook__changed_notebook.snap

```diff
@@ -0,0 +1,379 @@
+---
+source: crates/ruff_server/tests/notebook.rs
+expression: snapshot.query().as_notebook()
+---
+Some(
```

Comment:
Would it make sense in your view to instead render a visual representation of the notebook (by e.g. calling `to_string()` instead of snapshoting all internal fields? Snapshoting all fields seems brittle to internal changes of the notebook (which is exactly when we want to ensure that we didn't break anything, but the tests all change in that case, and become useless)

Reply:
I'm fine with that! It's probably easier to understand a diff like that, too.